src/main/resources/CanadianUni.json

```diff
@@ -0,0 +1,364 @@
+{
+    "canadianUniversities": [
```

Comment:
```suggestion
[
```
The list can be the top-level element of the file, I don't think we should nest it another level down like this. Just one less step for the code to do.
Also means removing L1 and L364 and reformatting all the indentation.

Reply:
Will do